compose-examples/lighthouse-only/README.md

```diff
@@ -1,13 +1,15 @@
-# Lighthouse standalone fullstack
+# Lighthouse node
 
 ## Requirements
 * Get to know [Lighthouse](https://lighthouse-book.sigmaprime.io/) a bit
 * Server with 4 (v)cpus & 8 gb memory & 100 gb storage
 
 ## Services
-* geth (beacon conntects to it to see deposits for validators)
+* [geth](https://github.com/ethereum/go-ethereum) (beacon conntects to it to see deposits for validators)
```

Comment:
conntects!!

Reply:
I think that's an outdated commit, this should be resolved and I can't find it anymore 🤔

Reply:
okey, got it